src/ORM/FieldType/DBInt.php

```diff
@@ -64,7 +64,13 @@ public function Nice()
     {
         return sprintf('%d', $this->value);
     }
-
+    
+    public function SpelledOut()
```

Comment:
```suggestion
    public function SpelledOut(): string
```
People will expect a string, but `NumberFormatter::format()` can return `false`.

This method should return an empty string in the event the `format()` method returns `false`.